selectors/example-selectors.md

```diff
@@ -180,39 +204,19 @@ Selector:
     sequence:
       ExploreFields:
         fields:
-          "Links"
+          "Name":
+            Matcher:
+              {}
+          "Data":
+            Matcher:
+              {}
+          "Links":
             ExploreAll:
               next:
                 ExploreFields:
                   fields:
-                    "multihash":
+                    "Hash":
                       ExploreRecursiveEdge
 ```
 
-
-If it's a file in some directory, you can also start at a deeper level:
```

Comment:
Aw I kinda like this one :/

It *is* problematic to make it both A) accurate to unixfsv1 and all its baroque details, and yet B) terse enough to be a good example, though.  I think it's a choose-one.

Reply:
That example was actually wrong. Directories are not represented as named keys (i.e. not as `{"mydir": {"subdir": …}}`, but with the names within the link itself. So you would need to have a conditional matcher which hasn't been specified yet. I tried to add it to the new version of the example I just pushed, but then the whole example didn't really make sense anymore (why would you have named links at all?), hence I left it out.